src/components/Data/Documents/Page.vue

```diff
@@ -412,12 +464,38 @@ export default {
           listViewType: filterManager.LIST_VIEW_MAP
         })
       )
+    },
+    listGeopoints(mapping, path = []) {
+      let attributes = []
+
+      for (const [attributeName, { properties }] of Object.entries(mapping)) {
+        if (properties) {
+          if (properties.lat && properties.lon) {
+            attributes = attributes.concat(path.concat(attributeName).join('.'))
+          }
+
+          attributes = attributes.concat(
+            this.listGeopoints(properties, path.concat(attributeName))
+          )
+        }
+      }
+
+      return attributes
+    },
+    loadCollectionInfo() {
```

Comment:
Since this method basically calls `getMappingDocument`, it essentially loads the ES mapping, right? I'm afraid the name `loadCollectionInfo` is a bit misleading...

Reply:
Also, didn't we already have a method that loads the mapping of the collection in this same page? If I'm correct, it was used to implement the basic-search field suggestions.

Reply:
It is the same method. Before the call to `getMappingDocument` was in the `mounted` method.  
I renamed it to `loadMappingInfo` because it loads the mapping and then find the geopoint list.